content/en/project/BLUP_Brain-Learning-Unicorn-Project/index.md

```diff
@@ -0,0 +1,233 @@
+
+## Personal presentation
+
+Welcome to this project dear unicorn student ! 
+
+<p align="center">
+<img src="https://media.giphy.com/media/CzQ9Kl1UIt8hG/giphy.gif">
+</p>
+<p> <font size="1.5"> Source: https://media.giphy.com/</font></p> 
+
+I am Elise, Ph.D. student in neurosciences at the UdeM since near to 4 years, and working on **the contribution of genetic to neurodevelopmental disorders** as autism. I don't really fit in a specific domain (genetic/cognitive neurosciences/psychology). I guess it is what we call a *unicorn student*? Currently, I am working with genetic data (Copy Number Variants), clinical phenotypes and doing a lot of statistics and graphs on R. But my initial formation was in cognitive neurosciences where I started to work with multimodal data (combination of Arterial Spin Labelling MRI data and Eye-tracking data). 
+
+Since I started my Ph.D., I never used MRI data nor python, and I am here to take a revenge on that. 
+
+**Skills:**
+
+- Data management (feed me with multimodal data plz)
+- Statistics
+- Debugging codes
+
+# Project definition
+
+## Background
+
+<p align="center">
+<img src="Slide_content/illustration_genetic.png">
+</p>
+
+<p> <font size="1">Source: Illustration inspired from freepik.com content and adapted on adobe illustrator</font></p> 
+
+Copy number variants (CNVs) are a family of structural variation of the chromosomes. They can be either a loss or a gain of a chromosome portion in comparison to a genome of reference. Sometimes, CNVs can be pathogenic, meaning that they are formally associated to neurodevelopmental or psychiatric disorders, such as autism spectrum disorders (ASD), Schizophrenia (SZ) or intellectual disability (ID). Such pathogenic CNVs have been associated to significant alterations of brain volume ([Modenato et al., 2020](https://www.medrxiv.org/content/10.1101/2020.04.15.20056531v1.full) ; [Martin-Brevet et al., 2018](http://www.sciencedirect.com/science/article/pii/S000632231831401X) ; [Maillard et al., 2015](https://www.nature.com/articles/mp2014145)) or connectivity ([Moreau et al., 2019](https://www.biorxiv.org/content/10.1101/862615v1.full)). Notably, there were common alterations of the insula volume when comparing structural brain alterations due to pathogenic CNVs and due to a neurodevelopmental disorder (e.g. ASD or SZ) ([Cauda et al., 2017](https://onlinelibrary.wiley.com/doi/abs/10.1002/aur.1759) ; [Goodkind et al., 2015](https://jamanetwork.com/journals/jamapsychiatry/fullarticle/2108651)).
+
+## Problematic 
+
+Can a model predict the genetic profile of an individual based on brain regions volumes?
+
+## Aims
+
+- **Main goal**: 
+
+This project aims to feed a machine learning model with brain volumes to predict if an individual is carrier of a potentially pathogenic CNV.
+
+<p align="center">
+<img src="Slide_content/BLUPproject.png">
```

Comment:
This image is missing. It should be added in the project repo and can be referenced as `<img src="BLUPproject.png">`. As long as the image is here, I'll be able to fix the link even if it's broken anyway.